zh/monitor-a-tidb-cluster.md

```diff
@@ -14,6 +14,11 @@ TiDB 通过 Prometheus 和 Grafana 监控 TiDB 集群。在通过 TiDB Operator
 
 在 [TiDB 集群监控](https://pingcap.com/docs-cn/stable/deploy-monitoring-services/)中有一些监控系统配置的细节可供参考。
 
+> **注意：**
+>
+> * 一个 TidbMonitor 只支持监控一个 TidbCluster。
+> * `spec.clusters[0].name` 需要配置为 TiDB 集群 TidbCluster 的名字。
```

Comment:
Please add this short introduction in the beginning of this doc. Otherwise users might not know what is TidbMonitor.
```
在 v1.1 及更高版本的 TiDB Operator 中，可以通过简单的 CR 文件（即 TidbMonitor）来快速建立对 Kubernetes 集群上的 TiDB 集群的监控。
```

Reply:
added in https://github.com/pingcap/docs-tidb-operator/pull/1002/commits/cf73140174f3b0cd4ba9722ef8b7efd872672e62